content/_data/changelogs/lts.yml

```diff
@@ -140,6 +140,77 @@
       message: Do not fail to write a log file just because something deleted the parent directory.
       issue: 16634
       pull: 2738
+- version: "2.46.1"
+  date: 2017-04-19
+  lts_predecessor: "2.32.3"
+  lts_baseline: "2.46"
+  lts_changes:
+    - type: rfe
+      message: Update the SSHD module from 1.7 to 1.8. The change disables obsolete Ciphers: AES128CBC, TripleDESCBC, and BlowfishCBC.
+      issue: 39805
+    - type: bug
+      message: Properties were not passed to Maven command by Maven build step when the Inject Build Variables flag was not set.
+      issue: 39268
+    - type: rfe
+      message: Enable the JNLP4 agent protocol by default.
```

Comment:
Hyperlink to https://github.com/jenkinsci/remoting/blob/master/docs/protocols.md#jnlp4-connect ?